interface/patient_file/summary/stats_full.php

```diff
@@ -77,6 +76,67 @@ function doeclick(id) {
     dlgopen('../problem_encounter.php?issue=' + encodeURIComponent(id), '_blank', 700, 400);
 }
 
+function getSelectionCheckBoxes(tableName) {
+    var result = [];
+    var table = document.getElementById(tableName);
+    for (var i = 0, row; row = table.rows[i]; i++) {
+        selBoxes = row.getElementsByClassName("selection-check");
+        if (selBoxes.length > 0 && selBoxes[0].id) {
+            result[result.length] = selBoxes[0];
+        }
+    }
+
+    return result;
+}
+
+function rowSelectionChanged(tableName) {
+    var deleteBtn = document.getElementById(tableName + "-delete");
+    if (!deleteBtn) {
+        return;
+    }
+
+    var selBoxes = getSelectionCheckBoxes(tableName);
+    for (var i = 0; i < selBoxes.length; i++) {
+        if (selBoxes[i].checked) {
+            deleteBtn.disabled = false;
+            return;
+        }
+    }
+
+    deleteBtn.disabled = true;
+}
+
+function headerSelectionChanged(checkBox, tableName) {
+    var selBoxes = getSelectionCheckBoxes(tableName);
+    for (var i = 0; i < selBoxes.length; i++) {
+        selBoxes[i].checked = checkBox.checked
+    }
+
+    rowSelectionChanged(tableName);
+}
+
+function deleteSelectedIssues(tableName) {
+    var selBoxes = getSelectionCheckBoxes(tableName);
+    var ids = ""
+    var count = 0;
+    for (var i = 0; i < selBoxes.length; i++) {
+        if (selBoxes[i].checked) {
+            if (count > 0) {
+                ids += ","
+            }
+            ids += selBoxes[i].id.split("_")[1]
+            count++
+        }
+    }
+
+    dlgopen('../deleter.php?issue=' + ids + '&csrf_token_form=' + <?php echo js_url(CsrfUtils::collectCsrfToken()); ?>, '_blank', 500, 450);
+}
+
+// Called by the deleter.php window on a successful delete.
+function imdeleted() {
+    refreshIssue('', '')
```

Comment:
Noted 2 bugs with this:
1. First is that if using a gui that only shows one issue category (Patient Summary screen ->Allergies->Edit button, which shows this:
![image](https://user-images.githubusercontent.com/278968/93296538-47d41500-f7a4-11ea-8757-59e8db4e6772.png)
When click delete button, it then shows all the issue categories

2. Get following popup after do a delete:
![image](https://user-images.githubusercontent.com/278968/93296695-95e91880-f7a4-11ea-87bb-232d9dff26ed.png)

Reply:
> Noted 2 bugs with this:
> 
> 1. First is that if using a gui that only shows one issue category (Patient Summary screen ->Allergies->Edit button, which shows this:

>    When click delete button, it then shows all the issue categories

Fixed.

> 2. Get following popup after do a delete:

I'm not seeing that on Chrome. I tried Firefox but pop-up windows don't launch even after I disabled the settings option to block pop-ups. So I can't reproduce it - I never use Firefox so I could use some help on this one.

Reply:
dlgopen can/will return a promise for a callback or to wait. Most likely, after calling dialog, code is falling through before dialog completes which can be unpredictable.
This could explain diff seen on diff browsers. consider using promise to ensure your app state for cleanup..

Reply:
> dlgopen can/will return a promise for a callback or to wait. Most likely, after calling dialog, code is falling through before dialog completes which can be unpredictable.
> This could explain diff seen on diff browsers. consider using promise to ensure your app state for cleanup..

Not sure I follow. This issue seems specific to Firefox and, from  my research so far, appears to be caused by re-loading the _status_full_ page via `location.reload()` from the `imdeleted` callback invoked via `deletet.php` -> `dlgclose`. I did this to re-display the table after the delete. I could confirm this if I could reproduce it but, as I mentioned, pop-up windows don't display on Firefox for me even tho I've unblocked pop-ups in the settings.

Reply:
This forms submit method most likely is POST where FF will ask to confirm resubmitting the form.(not unusual for Posted forms) All browsers use to have this behavior where now it looks like only FF. The only way I know to prevent this is to change the forms submit method to Get or refresh form in another way. 
You might call the form to be refreshed as if a new form instance instead of location.reload.

Reply:
> You might call the form to be refreshed as if a new form instance instead of location.reload.

yeah I can probably figure something out if I can reproduce it. Any idea how I can get pop-ups to display on FF? Otherwise I can blindly submit changes that work on Chrome and let others test FF. Or if someone else wants to try to fix it...

Reply:
This issue has been around since we quit using windows popups. You may not be seeing the problem due to a browser setting or such. Pop Up blocker will not prevent this particular notification from showing and if you haven't made any changes to the original opening of the form, cache and form state will be the same and you won't get the warning.

In this case, nothing is harmed by having this issue and it has been there for awhile so, just ignore and finish your work and i'll worry about fixing this.

Thanks for your contribution...

Reply:
ok - I read a post on StackOverflow that says using `window.location=window.location;` instead alleviates the popup. Works fine on Chrome, hopefully fixes it on FF.

Reply:
> Noted 2 bugs with this:
> 
> 1. First is that if using a gui that only shows one issue category (Patient Summary screen ->Allergies->Edit button, which shows this:

Fixed by setting `type='button'`.